### PR TITLE
Installed sentry

### PR DESCRIPTION
**Pull request by JanMikes**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,4 @@
 /build/
-node_modules/
```
Comment (TomasVotruba) [inline .gitignore:2]: Why this change? It's like `/vendor`

Comment (JanMikes) [reply .gitignore:2]: Duplication - it was there twice 😉

**Comment by JanMikes** [comment]
Blocked by Symplify/Symplify#1560
